app/models/course.rb

```diff
@@ -263,6 +263,10 @@ def training_module_ids
                                   .collect(&:training_module_ids).flatten
   end
 
+  def tracked_revisions
+    revisions.where.not(article_id: articles_courses.not_tracked.pluck(:article_id))
```

Comment:
It's a Lil confusing for me, there is a field named `tracked`, why not directly use the `tracked` and get `tracked revisions` directly?

Reply:
It is because articles courses isn't the one source of truth and so we exclude the untracked articles which are definitely available on articles_courses rather than including the other way around.

Reply:
@khyatisoneji this way, the default before any ArticlesCourses have been created is that all revisions will be tracked. ArticlesCourses get generated from the revisions, and we want to make sure there are no unexpected missing stats.